bedrock/products/templatetags/misc.py

```diff
@@ -254,3 +254,23 @@ def vpn_product_referral_link(ctx, referral_id="", page_anchor="", link_text=Non
     markup = f'<a href="{href}{page_anchor}" class="{css_class}" {attrs}>{link_text}</a>'
 
     return Markup(markup)
+
+
+@library.global_function
+def relay_mozmail():
```

Comment:
This is a fine way to go about it, but FYI this will also work in a template:

```jinja
{{ settings.RELAY_MAIL_DOMAIN }}
```

The `settings` object is available in all templates via a Django context processor.